docs/publishing.md

```diff
@@ -25,16 +25,18 @@ The files under scripts use [shelljs/shx](https://github.com/shelljs/shx) and
 [shelljs/shelljs](https://github.com/shelljs/shelljs) to write scripts in a
 portable manner across platforms.
 
+1. `git checkout -t origin release/vxx.yy.zz`
```

Comment:
I think Winston never updated this doc with the simplified steps.
But here it is.